Create de_DE.lang

TobiPeterG opened the pull request:
Add initial support for the German language on the 1.12 branch.
Everything is translated, I've tried to translated everything as good as possible, everything seems fine.

Have a nice day!

TobiPeterG committed:
Create de_DE.lang

Add initial support for the German language on the 1.12 branch.
Everything is translated, I've tried to translated everything as good as possible, everything seems fine.

Have a nice day!

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Sakuraholz
+tile.natura.overworld_logs2.sakura.tooltip=Blühende Kirsche
+tile.natura.overworld_logs2.hopseed.name=Hopfensaatholz
```
I think those might be called "Hop-Busch" in German, but I am not sure.

TobiPeterG replied:
I had a look at Wikipedia, it's called "Hopfenbusch" in German, so you were very close. ;)
https://de.wikipedia.org/wiki/Dodonaea_viscosa#Namensgebung

I'll correct it.

Vexatos replied:
That just says it's the literal translation, not what is actually used. I've seen "Hop-Busch" on the internet before, but I may not be right.

TobiPeterG replied:
Yeah, but I have literally no clue what a trivial German name for it may be.
If you wish, I'll name it "Hop-Busch", probably just a few people would notice a problem anyway.

But I'll do as you wish. :D

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+
+tile.natura.overworld_logs.maple.name=Ahornholz
+tile.natura.overworld_logs.maple.tooltip=Etwas süßlich
+tile.natura.overworld_logs.silverbell.name=Silberglöckchenholz
```
I think this one's name is "Schneeglöckchenbaum".

TobiPeterG replied:
Yeah, seems so, thanks. :)

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+
+tile.natura.overworld_leaves.maple.name=Ahornblätter
+tile.natura.overworld_leaves.maple.tooltip=Etwas süßlich
+tile.natura.overworld_leaves.silverbell.name=Glockenblumenblätter
```
This does not match up with the wood name above.

TobiPeterG replied:
You mean because of "Glockenblumen"?
Just noticed that as well as I was correcting "Silberglöckchen", fixed that.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_leaves.silverbell.tooltip=Silberne Glocken
+tile.natura.overworld_leaves.amaranth.name=Amaranthblätter
+tile.natura.overworld_leaves.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_leaves.tiger.name=Tigerblätter
```
This should be "Tigerholzblätter" since the name of the tree is "Tigerholz".

TobiPeterG replied:
Well, I also though about that, but we say e.g. "Eichenblätter" and not "Eichenholzblätter".
That's way I went with this solution. O.o

Vexatos replied:
"Eichenblätter" comes from the plant called "Eiche". The plant here is called "Tigerholz" so this has to be called "Tigerholzblätter".

TobiPeterG replied:
OK, I'll correct it.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_logs2.willow.tooltip=Der Weinende
+tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Sakuraholz
```
The name of this plant is "Japanische Blütenkirsche", but I think "Blütenkirsche" would suffice here.

TobiPeterG replied:
Uh, you're right, I'll correct it.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_sapling.silverbell.tooltip=Silberne Glocken
+tile.natura.overworld_sapling.amaranth.name=Amaranthsetzling
+tile.natura.overworld_sapling.amaranth.tooltip=Herz aus Holz
+tile.natura.overworld_sapling.tiger.name=Tigerbaumsetzling
```
Perhaps this should be called "Tigerholzsetzling".

TobiPeterG replied:
Yeah, that may be an alternative, but I tried to follow the Minecraft name scheme, and e.g. the Jungle tree sapling is called "Tropenbaumsetzling", most of the others are just "Eichensetzling", but "Tigersetzling" is a bit weird, which is why I went with "Tigerbaumsetzling".
What do you mean?

Vexatos replied:
See https://github.com/progwml6/Natura/pull/496#discussion_r426647685. The correct name for this plant is "Tigerholz", the "holz" is just part of its name and does not refer to the material.

TobiPeterG replied:
OK.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_sapling2.hopseed.name=Hopfensaatsetzling
+tile.natura.overworld_sapling2.hopseed.tooltip=Aufgestiegener Fehler
+
+tile.natura.redwood_logs.bark.name=Rotholzrinde
```
redwood in German is called "Mammutbaum".

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_logs.darkwood.name=Dunkelholz
+tile.natura.nether_logs.darkwood.tooltip=Netherbaum
+tile.natura.nether_logs.fusewood.name=Zündholz
+tile.natura.nether_logs.fusewood.tooltip=Netherbaum\nExplodierende Persönlichkeit
```
"Explosive"?

TobiPeterG replied:
Yeah, sounds better. Thanks. :)

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+
+tile.natura.nether_logs.ghostwood.name=Geisterholz
+tile.natura.nether_logs.ghostwood.tooltip=Netherbaum\nBleich wie ein Gespenst
+tile.natura.nether_logs.darkwood.name=Dunkelholz
```
Maybe "Düsterholz" or "Finsterholz" would be a cool name.

TobiPeterG replied:
Good idea, I like "Finsterholz" a bit more I think. :)

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_tainted_soil.tainted_farmland_dry.name=Verdorbener Boden
+tile.natura.nether_tainted_soil.tainted_farmland_heated.name=Verdorbener Boden
+
+tile.natura.nether_heat_sand.name=Hitzesand
```
Maybe some nicer sounding name would fit here. Ideas from me would be "Glühsand" or "Schwelsand".

TobiPeterG replied:
Uh yeah, "Schwelsand" is nice, thanks. :)

TobiPeterG replied:
I've also corrected "Hitzespinne", "Hitzeglas", .. to "Schwel..."

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_berrybush_blightberry.tooltip=Ein bisschen böse\nKillerheilung
+tile.natura.nether_berrybush_duskberry.name=Waldbeerenbusch
+tile.natura.nether_berrybush_duskberry.tooltip=Ein bisschen böse\nUnsichtbare Nacht
+tile.natura.nether_berrybush_skyberry.name=Himmelsbeerenbusch
```
"Himmelsbeerbusch" would be more correct.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+
+tile.natura.nether_berrybush_blightberry.name=Brombeerstrauch
+tile.natura.nether_berrybush_blightberry.tooltip=Ein bisschen böse\nKillerheilung
+tile.natura.nether_berrybush_duskberry.name=Waldbeerenbusch
```
"Waldbeerbusch" would be more correct. Perhaps change the name to "Düsterbeerbusch" and the berries to "Düsterbeere"? That sounds like a nice name for duskberries. Or maybe "Finsterbeere".

TobiPeterG replied:
OK, I'll go for "Düsterbeere" so that it's not confused with/set in relation to the "Finsterholz".

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_berrybush_duskberry.tooltip=Ein bisschen böse\nUnsichtbare Nacht
+tile.natura.nether_berrybush_skyberry.name=Himmelsbeerenbusch
+tile.natura.nether_berrybush_skyberry.tooltip=Etwas böse\nLangsames Springen
+tile.natura.nether_berrybush_stingberry.name=Stechbeerenbusch
```
"Stechbeerbusch" would be more correct.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_berrybush_blightberry.name=Brombeerstrauch
+tile.natura.nether_berrybush_blightberry.tooltip=Ein bisschen böse\nKillerheilung
+tile.natura.nether_berrybush_duskberry.name=Waldbeerenbusch
+tile.natura.nether_berrybush_duskberry.tooltip=Ein bisschen böse\nUnsichtbare Nacht
```
This would be "Sichtbare Nacht" instead.

TobiPeterG replied:
Yeah, that would be the litereral translation, but since it has the effect night sight and blindness, I found "Unsichtbare Nacht" better fitting.
But if you wish, I'll  change it.

Vexatos replied:
I actually like your interpretation, keep it.

TobiPeterG replied:
OK, thanks. :)

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+
+tile.natura.nether_heat_sand.name=Hitzesand
+
+tile.natura.nether_berrybush_blightberry.name=Brombeerstrauch
```
This is wrong. A suggestion for translating blightberry might be "Leidbeere", "Peinbeere", "Fahlbeere" or something similar. I personally like the last one of these.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_berrybush_skyberry.name=Himmelsbeerenbusch
+tile.natura.nether_berrybush_skyberry.tooltip=Etwas böse\nLangsames Springen
+tile.natura.nether_berrybush_stingberry.name=Stechbeerenbusch
+tile.natura.nether_berrybush_stingberry.tooltip=Ein bisschen böse\nSchlag wie ein Lastwagen
```
This should not be translated literally. I suggest something along "Triff da, wo es weh tut"

TobiPeterG replied:
Yeah, I know, but I found that translation kinda funny so I thought it might be fitting.
But I'll change it.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_berrybush_blueberry.tooltip=Herb und blau\nWächst in gemäßigten Zonen
+tile.natura.overworld_berrybush_blackberry.name=Brombeerstrauch
+tile.natura.overworld_berrybush_blackberry.tooltip=Süß und schwarz\nWächst in feuchten Gebieten
+tile.natura.overworld_berrybush_maloberry.name=Maloberrybusch
```
This could just be "Malobeerbusch".

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+item.natura.edibles.blightberry.tooltip=Ein bisschen böse\nKillerheilung
+item.natura.edibles.duskberry.name=Duskberry
+item.natura.edibles.duskberry.tooltip=Ein bisschen böse\nUnsichtbare Nacht
+item.natura.edibles.skyberry.name=Himmelsbeere
```
"Himmelbeere" would be much nicer and fit with the names above.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+
+item.natura.edibles.blightberry.name=Preiselbeere
+item.natura.edibles.blightberry.tooltip=Ein bisschen böse\nKillerheilung
+item.natura.edibles.duskberry.name=Duskberry
```
See comments above.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+item.natura.edibles.maloberry.name=Malobeere
+item.natura.edibles.maloberry.tooltip=Sättigend!
+
+item.natura.edibles.blightberry.name=Preiselbeere
```
This one is definitely wrong. See above.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+item.natura.edibles.skyberry.name=Himmelsbeere
+item.natura.edibles.skyberry.tooltip=Etwas böse\nLangsames Springen
+item.natura.edibles.stingberry.name=Stechbeere
+item.natura.edibles.stingberry.tooltip=Ein bisschen böse\nSchlag wie ein Lastwagen
```
See above.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+item.natura.empty_bowls.darkwood_bowl.name=Leere Dunkelholzschale
+item.natura.empty_bowls.fusewood_bowl.name=Leere Zündholzschale
+
+item.natura.soups.ghostwood_mushroomstew.name=Champignoneintopf
```
Just call it "Pilzeintopf".

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+item.natura.imp_armor_leggings.name=Koboldhauthose
+item.natura.imp_armor_boots.name=Koboldhautstiefel
+
+item.natura.seed_bags.wheat_seed_bag.name=Weizensamenbeutel
```
"Weizensaatbeutel" might be better.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+#    Tools     #
+################
+
+item.natura.flint_and_blaze.name=Feuerstein und Lohe
```
Flint & Steel are translated as "Feuerzeug", how about calling this "Lohenfeuerzeug"?

TobiPeterG replied:
Yeah, to be honest I wasn't sure how t translate it properly, but you're right.

BTW. I had not really a clue what "Ascended Glitch" was refering to and just translated it to "Aufgestiegener Fehler". But I don't think that's the best solution, but couldn't think of a better one instantly. May you have an idea?

Vexatos replied:
Well it's a reference so it's really hard to translate. It's always valid to just pick something completely different as long as it is somewhat related to the thing.

TobiPeterG replied:
OK, thanks. :)

Vexatos reviewed (changes requested):
Thanks for updating this, but there are a few things you might want to look at. I didn't comment multiple times if the same renaming should appear in multiple places, but it obviously should stay consistent. Note that many of them are suggestions and not corrections; let me know what you think.

TobiPeterG committed:
Update de_DE.lang

I hopefully fixed all issues, tried to update every corresponding string.

Please tell me, if I forgot something! :)

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -24,58 +24,58 @@ tile.natura.overworld_logs2.willow.name=Weidenholz
 tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
 tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
-tile.natura.overworld_logs2.sakura.name=Sakuraholz
+tile.natura.overworld_logs2.sakura.name=Blütenkirschenholz
```
This should be "Blütenkirschholz".

TobiPeterG replied:
Umm my commit included that already. O.o :D

TobiPeterG replied:
Just saw the issue. sorry.
Corrected it.

TobiPeterG committed:
Update de_DE.lang

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -24,7 +24,7 @@ tile.natura.overworld_logs2.willow.name=Weidenholz
 tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
 tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
-tile.natura.overworld_logs2.sakura.name=Blütenkirschenholz
+tile.natura.overworld_logs2.sakura.name=Blütenkirschholz
```
This should have been applied for _all_ references to sakura further down, too.

TobiPeterG replied:
ofc, sorry.

TobiPeterG committed:
Update de_DE.lang

TobiPeterG commented:
So is there anything more I can do for you? ;)

progwml6 commented:
@Vexatos please leave a review when you feel comfortable with this being merged

Vexatos reviewed (approved):
I still think hopseed should have a different tooltip, Currently it's translated literally which doesn't really work; but that is arguably minor. Apart from that, this translation is a really good update to my old one, and in some places much better. The only important thing is that the joke in "fusewood" was kept ;) It's nice to see someone with decent translation skills for once.

TobiPeterG commented:
> I still think hopseed should have a different tooltip, Currently it's translated literally which doesn't really work; but that is arguably minor. Apart from that, this translation is a really good update to my old one, and in some places much better. The only important thing is that the joke in "fusewood" was kept ;) It's nice to see someone with decent translation skills for once.

Uh I can change it to Hop-Busch real quick, no problem.

Thanks. :)

Vexatos commented:
I was more talking about its tooltip, which, as you commented earlier, doesn't really work as it currently is.

TobiPeterG commented:
> I was more talking about its tooltip, which, as you commented earlier, doesn't really work as it currently is.

Ah OK, sorry, yeah, that's really a problem.
Other language files couldn't really help me either there. O.o
EDIT: For example it's also just "Aufgestiegener Fehler/Glitch" in the Russian file.

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_logs2.sakura.name=Blütenkirschholz
+tile.natura.overworld_logs2.sakura.tooltip=Blühende Kirsche
+tile.natura.overworld_logs2.hopseed.name=Hopfenbuschholz
+tile.natura.overworld_logs2.hopseed.tooltip=Aufgestiegener Fehler
```
I suggest just going with something completely unrelated, but still referring to the plant. Some stuff I just came up with is for example "Tiefliegend" since it's a low-growing plant; something like that. Use your imagination.

TobiPeterG replied:
yeah, that'd be possible.
I just read the Wikipedia article to this plant and saw that it's sticky.
Maybe we could combine that with your idea and change it to "Klein und klebrig".
I think that'd fit quite well, what do you think?

Vexatos replied:
That sounds nice enough.

TobiPeterG replied:
OK, thank you! :)

Vexatos commented on src/main/resources/assets/natura/lang/de_DE.lang:
```diff
@@ -0,0 +1,582 @@
+tile.natura.overworld_logs2.eucalyptus.name=Eukalyptusholz
+tile.natura.overworld_logs2.eucalyptus.tooltip=Das rosa Holz
+tile.natura.overworld_logs2.sakura.name=Blütenkirschholz
+tile.natura.overworld_logs2.sakura.tooltip=Blühende Kirsche
```
On that note, this one is a bit redundant since it's already in the name of the plant, maybe change this too. One idea I had is just "Herz­er­wär­mend" since it's pretty to look at.

TobiPeterG replied:
Yes, that's good, I, again, just read the Wikipedia article to find an idea and found out that in Japan, the sakura stands for beauty, departure and transience.
So I thought we could go "Schön, aber vergänglich", but it isn't transient in Minecraft, so that doesn't really fit. We may say "Schön und unvergänglich" to draw attention to that issue.
We could also make it even more confusing and go for "Schöner Aufbruch". But I don't know if that's wanted.

Vexatos replied:
I like "Schön und unvergänglich".

TobiPeterG replied:
OK, I'll create a commit to fix both issues. :)

TobiPeterG committed:
Update de_DE.lang

enhance hopbush and sakura tooltips

TobiPeterG committed:
Update de_DE.lang

refix sakura (forgot to save changes to my local file last time)
